ui/assets/css/style.css

```diff
@@ -57,11 +55,17 @@ nav ul li{
     float: left;
 }
 nav ul li a{
-    text-decoration: none;
+    font-weight: bold;
+    border: 1px solid #fff;
+    padding: 5px 20px;
+    color: #111;
+    margin: 2px;
+    transition: 0.6s ease;
+    /* text-decoration: none;
```

Comment:
Please delete this comment lines of code also.

Reply:
Deleted